app/main/helpers/frameworks.py

```diff
@@ -108,7 +108,10 @@ def get_formatting_errors(answers):
     errors_map = {}
     if not re.match(r'^(\S{9}|\S{12})$', answers.get('SQ1-1h', '')):
         errors_map['SQ1-1h'] = 'invalid_format'
-
+    if not re.match(r'^[^@^\s]+@[^@^\.^\s]+(\.[^@^\.^\s]+)+$', answers.get('SQ1-1o', '')):
```

Comment:
Looking at the code it's not immediately obvious that this is email validation — I'd struggle to find this part if I were searching. I think saving the regex pattern to a named constant could help here a bit.